griptape/templates/memory/meta/meta_memory.j2

```diff
@@ -0,0 +1,8 @@
+{% if meta_memories|length > 0 %}
```

Comment:
Does this need to be `{% if meta_memories and meta_memories|length > 0 %}`? Similar to `rulesets.j2`.

Reply:
It can actually be just `{% if meta_memories %}`.